crud.py

```diff
@@ -0,0 +1,49 @@
+from datetime import date
+from sqlalchemy.orm import Session
+
+from db import models
+from schemas import AuthorCreate, BookCreate
+
+
+def create_author(db: Session, author: AuthorCreate):
+    db_author = models.Author(
+        name=author.name,
+        bio=author.bio,
+    )
+    db.add(db_author)
+    db.commit()
+    db.refresh(db_author)
+
+    return db_author
+
+
+def get_authors(db: Session, skip: int = 0, limit: int = 10):
+    return db.query(models.Author).offset(skip).limit(limit).all()
+
+
+def get_author_by_id(db: Session, author_id: int):

```

Comment:
Consider adding type hints for the return type of the function, e.g., `-> models.Author | None`.